src/main/java/gyro/azure/storage/PolicyBaseBlob.java

```diff
@@ -0,0 +1,93 @@
+package gyro.azure.storage;
+
+import com.microsoft.azure.management.storage.DateAfterModification;
+import com.microsoft.azure.management.storage.ManagementPolicyBaseBlob;
+import gyro.azure.Copyable;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Updatable;
+import gyro.core.validation.ValidationError;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PolicyBaseBlob extends Diffable implements Copyable<ManagementPolicyBaseBlob> {
+    private Integer deleteDays;
+    private Integer tierToArchiveDays;
+    private Integer tierToCoolDays;
+
+    /**
+     * Days after which blob object to be deleted. At least one of 'delete-days' or 'tier-to-cool-days' or 'tier-to-archive-days' is required.
```

Comment:
`Days after which a blob object will be deleted.` sounds a little better I think.